### PR TITLE
Code improvements

### PR DESCRIPTION
**Pull request by 654wak654**
**When merged this pull request will:**
- Add the use of the `params` command in some places.
- Add the use of lazy evaluation in some places.
- Remove some excessive whitespace.
- Add newlines at the end of files for git.
- Add a link to this repository in the CfgPatches.
- Slightly change the forEach loop in `fn_startCaptureLoop.sqf` so that filtering is done only once.

### CODE REVIEW

**Review by JamieGoodson — CHANGES_REQUESTED** [review]
Great optimizations. Before I approve, can you confirm if this has been tested (i.e. capture and playback, including events, are both working correctly). Also, please could two capture tests be performed (one with these changes, one without) to confirm that frame capture times are equal/reduced. Run @ocap as client and enable debug mode to see frame capture times in systemChat.

**Comment by 654wak654** [comment]
Was able to test this multiple times recently. I didn't notice `startCaptureLoop` behaving any different, but the play sessions have been with low amounts of players.

**Comment by JamieGoodson** [comment]
Thanks for making these changes. I've recently started working on OCAP again. Since the codebase has changed a fair amount now, I will close this PR. However, I'm still going to pull these optimisations through 👍

**Pull request closed without merging**
